packages/rum-core/src/boot/preStartRum.ts

```diff
@@ -96,6 +98,10 @@ export function createPreStartStrategy(
       return
     }
 
+    // Instrumuent fetch and XHR to track network requests
+    initFetchObservable()
+    initXhrObservable(configuration)
```

Comment:
**💬 suggestion:** ‏move this a bit below in the function, after all the early returns

**💬 suggestion:** in the comment, ‏add more details on the motivation why we're doing this here instead of lazily when starting `requestCollection`

**💬 suggestion:** ‏add a unit test to ensure `fetch` and `xhr` are instrumented when running `init()` even without consent